lib/dateTime.js

```diff
@@ -738,6 +738,18 @@ export function getDateTimeDescriptor() {
 			weekendStartDay = 4;
 			weekendEndDay = 5;
 			break;
+		case 'cy':
```

Comment:
Most of this I grabbed from the monolith's LOCALE_CULTURE table and lang terms, the rest from the CLDR.